android/build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.1'
+        classpath 'com.google.gms:google-services:4.3.8'
```

Comment:
Looks like we have a new release recently: https://mvnrepository.com/artifact/com.google.gms/google-services/4.3.9

Reply:
Use latest version of google-services android package. Also remove unnecessary dependency in plugin project.	110a4a7	Ben Butterworth <24711048+ben-xD@users.noreply.github.com>	10 Aug 2021 at 11:23